ansible/roles/common_installs/tasks/main.yml

```diff
@@ -104,3 +105,11 @@
   pip:
     name: virtualenv-clone
     version: 0.2.6
+
+- name: Install npm (only) a second time
```

Comment:
comment would be useful explaining why

Reply:
Oops, I meant to either do the above or this. When I tested just now, I thought I was testing without this task. Let me run again without it and see if this task is necessary.